src/color/Color.ts

```diff
@@ -310,14 +251,14 @@ export class Color {
    * @see http://http://www.w3.org/TR/css3-color/#hsl-color
    */
   static sourceFromHsl(color: string): TRGBAColorSource | undefined {
-    const match = color.match(reHSLa);
+    const match = color.match(reHSLa());
     if (!match) {
       return;
     }
 
     const h = (((parseFloat(match[1]) % 360) + 360) % 360) / 360,
```

Comment:
i wonder... this seems correct and i noticed was off.
I m trying to think of some non obvious value that could break.
But i prefer to discover in this case and in case fix later because this seems wrong

Reply:
Ahhh I see.
What if the value is < -360. Now is fails, before it was safe.
Please revert this.
I didn't think of that.

Reply:
Yeah that is fine, we discovered.